flocker/node/agents/_logging.py

```diff
@@ -0,0 +1,66 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+from eliot import Field, ActionType, MessageType
+
+# Begin: Helper datastructures to log all
+# IBlockDeviceAPI calls from AWS storage driver using Eliot.
+
+# An OPERATION is a list of:
+# IBlockDeviceAPI name, positional arguments, keyword arguments.
+OPERATION = Field.forTypes(
+    u"operation", [list],
```

Comment:
Is ``list`` a valid ``Field`` type?
If the supplied list might contain other complex types, then Eliot will need to know how to serialise those.
I'm thinking of a situation where a caller has supplied the wrong argument types to ``attach_volume``. In which case the list might be  `['attach_volume', BlockDeviceVolume()...]`. How will Eliot know how to serialise `BlockDeviceVolume` ? Does it just call `repr`? Or will it blow up when our `stdout` log observer attempts to serialise that object?
@itamarst will know.